News: add #45 (2019-05-07)

harding opened the pull request.

jnewbery commented on _posts/en/newsletters/2019-05-07-newsletter.md:
```diff
@@ -0,0 +1,273 @@
+      PSBTs; `analyzepsbt` to tell the user what they next need to do
+      with the PSBT; and `utxoupdatepsbt` to add necessary information
+      to a PSBT from a node's UTXO set.  Additionally, new [PSBT
+      documentation][] is provided---it's based on the PSBT section from
```
nit: 'provided---it's based on...' seems a bit disjointed. How about "provided, which is based on ..."?

Mine8801 replied:
https://github.com/bitcoinops/bitcoinops.github.io/pull/139#discussion_r281443049

jnewbery commented on _posts/en/newsletters/2019-05-07-newsletter.md:
```diff
@@ -0,0 +1,273 @@
+
+    - *Initial RPC support for output script descriptors:* Bitcoin
+      wallets need a way to find all the transactions on the block chain
+      that pay their keys.  This is easy if the wallet only supports one
```
nit: how about 'pay to addresses controlled by their keys' or 'pay to outputs controlled by their keys' rather than 'pay their keys'?

harding replied:
I changed s/their keys/the user/ because I don't think this needs to be too specific given that the next sentence gets into details about scriptPubKey.  However, any of these versions is fine by me if you want to change it.  (I slightly prefer "outputs controlled by their keys" since we later talk about P2PK, which isn't strictly an address.)

harding replied:
Eh, on doing a final proofread, I changed this even more.  In any case, please feel free to edit as you see fit at merge time.

jnewbery commented on _posts/en/newsletters/2019-05-07-newsletter.md:
```diff
@@ -0,0 +1,273 @@
+      newsletter---tests are rarely news unless something goes
+      wrong---so we wanted to take this opportunity to remind readers
+      that testing remains an active and important part of Bitcoin Core
+      development.
```
:heart_eyes:

jnewbery commented on _posts/en/newsletters/2019-05-07-newsletter.md:
```diff
@@ -0,0 +1,273 @@
+  only creates a clearer division between layers in the system's code
+  but it also prepares for future improvements in the peer-handling code
+  that will allow it to be smarter about banning peers based on multiple
+  criteria.
```
Great summary!

Mine8801 replied:
Hi

moneyball commented on _posts/en/newsletters/2019-05-07-newsletter.md:
```diff
@@ -0,0 +1,273 @@
+      mentioned in the *news* section of [Newsletter #40][], the release
+      notes announce the project's intention to switch to bech32 sending
+      addresses by default in either the next major version (0.19,
+      [expected around January 2020][0.19 release schedule]) or the
```
This has been updated to November 2019. https://github.com/bitcoin/bitcoin/issues/15940#issuecomment-489921644

harding commented:
Force pushed an edit that hopefully addresses all feedback and which also makes a few minor edits based on my final proofread.  Thanks @jnewbery and @moneyball!

practicalswift commented:
Yet another excellent newsletter! Thanks!

ACK 6d01142ad2bed841f8ce5cd9e340da2aff383120

jnewbery commented:
utACK 6d01142ad2bed841f8ce5cd9e340da2aff383120

Thanks Dave!
